### PR TITLE
accept inavlid ssl cert for development

### PR DESCRIPTION
**Pull request by mikkeschiren**
A suggestion to except invalid ssl cert in curl request for development, I think it could help out people for some functionality.

### CODE REVIEW

**Comment by sgiehl** [comment]
Guess would make sense to add a default value to global.ini.php with a comment what it's good for.
See https://github.com/matomo-org/matomo/blob/35467f54cd3e1dae889d88ae4f80d4626f9127ed/config/global.ini.php#L177-L188

**Comment by Findus23** [comment]
Honestly I think this option should either only work when development mode is enabled or show a permanent, non-dismissable warning in Matomo (or at least the system check).
Otherwise it would be far to easy to accidently undermine a huge part of the security.

**Comment by mikkeschiren** [comment]
Yeah agree - this should only work if Development module is active, and show a warning at system check - my goal with this first version on the pull request is to get the feedback. There are some problems with some development tools for some setups if you can not bypass invalid certs (local development mostly), but like said - it could be a security risk.

File: core/Http.php [hunk 1]
```diff
@@ -96,8 +96,13 @@ public static function sendHttpRequest($aUrl,
         // create output file
         $file = self::ensureDestinationDirectoryExists($destinationPath);
 
+        $acceptInvalidSslCertificate = false;
+        if (isset(Config::getInstance()->Development['accept_invalid_ssl'])) {
```
Comment (tsteur) [inline core/Http.php:100]: @mikkeschiren could you add a default value to the `config/global.ini.php` for `accept_invalid_ssl`? And also add `Development::isEnabled()` to the check?

Comment (tsteur) [reply core/Http.php:100]: @mikkeschiren any chance you could check out my comment above and add the default value?

**Comment by tsteur** [comment]
@mikkeschiren I'll close this PR for now as we haven't heard from you in a while but will be happy to reopen should you be keen to follow up on the other comments. Simply let us know and we'd be happy to review things again.

**Pull request closed without merging**